src/components/calcite-dropdown-group/calcite-dropdown-group.tsx

```diff
@@ -68,7 +67,7 @@ export class CalciteDropdownGroup {
     this.calciteDropdownGroupRegister.emit({
       items: this.items,
       position: this.groupPosition,
-      groupId: this.dropdownGroupId,
+      group: this.el as HTMLCalciteDropdownGroupElement,
```

Comment:
This can be avoided if el is typed as `HTMLCalciteDropdownGroupElement`.

Reply:
Yeah, agreed - should this be something we change across all components? I think only a few are currently typed as their own `@Element() el: HTMLCalcite--Element`, everything else is `@Element() el: HTMLElement`

Reply:
Sounds like it should. It would be more accurate to set it to its correct type.